src/main/java/com/kodilla/ecommercee/domains/User.java

```diff
@@ -32,10 +32,11 @@ public User(String username, String status, Long userKey) {
     @Getter
     @Setter
     @Column(name = "STATUS")
-    private String status;
+    private boolean status = true;
```

Comment:
jeżeli stosujemy boolean to chyba lepiej zastosować inną nazwe zmiennej, coś w stylu "blocked".

Reply:
Nazwa zgodnie z tym, co mielismy podane od poczatku projektu, wiec zostawiam.

Reply:
ok, zmienione na int i przypisany 1, by po utworzeniu bylo 1, a po blockUser 0